index.html

```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Music festival</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/86098cddac.js" crossorigin="anonymous"></script>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700;900&display=swap" rel="stylesheet">
+    <script defer src="script.js"></script>
+</head>
+<body>
+    <section id="first-page">
+        <header>
+            <div class="burger-button">
+               <div class="menu-line"></div>
+               <div class="menu-line"></div>
+               <div class="menu-line"></div>
+            </div>
+            <div class="header-desk">
+                <ul class="top-bar">
+                    <li><i class="fa-brands fa-facebook-f"></i></li>
+                    <li><i class="fa-brands fa-twitter"></i></li>
+                    <li>English</li>
+                    <li>My Page</li>
+                    <li>Logout</li>
+            </div>
+            <nav class="nav-bar">
+                <div class="logo">
+                    <img src="images/logo_square_large_print.webp" alt=""></img>
+                </div>
+                <ul class="nav-list">
+                  <li class="nav-item">
+                    <a  href="index.html">Home</a>
+                  </li>
+                  <li class="nav-item">
+                    <a href="about.html">About</a>
+                  </li>
+                  <li class="nav-item">
+                    <a  href="">Program</a>
+                  </li>
+                  <li class="nav-item">
+                    <a href="">Join</a>
+                  </li>
+                  <li class="nav-item">
+                    <a  href="">Sponsors</a>
+                  </li>
+                  <li class="nav-item">
+                    <a  href="#">News</a>
+                  </li>
+                  <li class="nav-item">
+                    <a href="#">CC compaign</a>
+                  </li>
+                </ul>
+            </nav>
+        </header>
+        <div class="headline">
+            <h3 class="hello">"Hello! Sharing World!"</h3>
+            <h1 class="title">The Event Listener Music Festival 2015</h1>
+            <div class="intro-box">
+                <p>A joyful celebration believing in the value of openness and sharing, creating a positive change with people from over 80 countries is taking place in October, in Korea.</p>
+            </div>
+            <h2 class="dates">2015.10.15(THU)~16(FRI)</h2>
+            <p>@Gwangalli Beach, Busan</p>
+        </div>
+    </section>
+    <section id="main-program">
+        <h3 class="including">Including</h3>
+        <div class="include-container">
+        <ul class="include-list">
+            <li><i class="fa-sharp fa-solid fa-music"></i></li>
```

Comment:
- [ ] It could be better if you add some `padding` and the `background image` of the section to stick the design 

> | | Figma template | | Your project | |
> |-|---------|-|--------|-|
> | |![Capture1](https://user-images.githubusercontent.com/57408419/234875097-078a33fa-2e5b-4e98-ad03-a4d09b7db10c.JPG)| |![Capture](https://user-images.githubusercontent.com/57408419/234874957-db01fde5-e3d8-453f-a70c-e09d6b324ce8.JPG)| |